pandas/core/algorithms.py

```diff
@@ -1102,6 +1102,37 @@ def _get_score(at):
         return result
 
 
+def check_percentile(q: Union[float, Iterable[float]]) -> np.ndarray:
```

Comment:
would this make more sense in validators?

Reply:
I think that's reasonable

Reply:
Should I shift it to utilts/_validators.py ?

Reply:
Yea let's do that

Reply:
Would renaming it to validate_percentile be better?